index.html

```diff
@@ -698,12 +704,49 @@ <h3>
             </p>
           </li>
         </ol>
+        <p data-dfn-for="PerformanceResourceTiming">
+          The <dfn>renderBlockingStatus</dfn> getter steps are to return the
+          <a data-for="PerformanceResourceTiming">render-blocking status</a>
+          for <a>this</a>.
```

Comment:
So this is rendered as `The renderBlockingStatus getter steps are to return the render-blocking status for this.`

Shouldn't be something like `The renderBlockingStatus getter steps are to return the render-blocking status for this's timing info's render-blocking and the relevant global object for this.`?

Reply:
The enum's value is determined [here](https://pr-preview.s3.amazonaws.com/abinpaul1/resource-timing/pull/327.html#sec-render-blocking-status-types) using `timing info's render-blocking` and I thought `getter` would have to just return that value.

Do I have to draft it as `return the render-blocking status for this's timing info's render-blocking and the relevant global object for this` to signify that the `timing info's render-blocking` is used to set the value of enum that is returned?

Reply:
I see, I missed that enum part. 

I am not too sure...Just because I expect the enum should just define the meaning of each value, not involves in effectively getting the value from `timing info`. 

Maybe I am just picky here and causing unnecessary troubles.

@noamr can confirm..

Reply:
Yes it's odd to define a getter in the enum.
The enum should describe the meaning, and the getter steps should be to get the enum value based on the boolean value as @sefeng211 said.

Reply:
Cool! I've updated it to do that. Thanks!